src/index.html

```diff
@@ -17,16 +17,34 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <label
+      class="lab-first"
+      data-qa="big"
+    >
+      <img
+        alt="search-icon"
+        src="images/Search.svg"
+      />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <input
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </label>
+    <label
+      class="lab-second"
+      data-qa="small"
+    >
+      <img
+        alt="search-icon"
```

Comment:
The alt attribute should describe the image content. 'search-icon' is not descriptive enough. Consider using a more descriptive alt text, such as 'magnifying glass icon for search'.